### PR TITLE
Add Fakturoid

### PR DESCRIPTION
**Pull request by marketauml**

### CODE REVIEW

File: entries/f/fakturoid.cz.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Site Name": {
```
Comment (hkamran80) [inline entries/f/fakturoid.cz.json]:
```suggestion
  "Fakturoid": {
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Please add the SVG from #7850 to this PR.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Use this icon and save it as `img/f/fakturoid.cz.svg`:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 201.6 195.8"><path fill="#1a1919" d="M100.8 0C45.1 0 0 35.8 0 80v35.8c0 44.2 45.1 80 100.8 80 55.7 0 100.8-35.8 100.8-80V80c0-44.2-45.1-80-100.8-80z"/><path fill="#fff" d="M132 57.5C107.4 57.5 99 71 99 81v7.1h.9C103.2 84 112.2 78 128.2 78c13.3 0 19.3 10 19.3 22.6h-7.8c-32.8 0-44.4 12-44.4 28 0 16.3 14.1 30.3 36.1 30.3h4.7c20.6 0 35-13.6 35-33.1v-21.3c0-32-18.5-47-39.2-47zm8.4 59.4h8.2v9c0 9.4-4.8 14.2-14.8 14.2-10.1 0-16-4.3-16-11.5 0-7.6 5-11.7 22.6-11.7zM68 118.5H58v38.8h-9.3c-7.4 0-14-5.7-14-13.8V82.3c0-18.9 8.9-42.6 35.2-48.4 26.7-6 34.2 7.1 34.2 7.1s-16.7-.2-29.8 10A45.8 45.8 0 0 0 58 84.2v18a59.5 59.5 0 0 0 34.2-4.1s-1.1 20.4-24.2 20.4z"/></svg>
```

File: entries/f/fakturoid.cz.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "domain": "fakturoid.cz",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://www.fakturoid.cz/podpora/nastaveni/dvoufazove-overeni",
+    "categories": [
+      "finance"
+    ]
```
Comment (Carlgo11) [inline entries/f/fakturoid.cz.json:13]:
```suggestion
    ],
    "regions": [
      "cz"
    ]
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by marketauml** [comment]
> Please add the SVG from #7850 to this PR.

Okay, done, please, check that

**Review by Carlgo11 — APPROVED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**